src/checkoutservice/main.go

```diff
@@ -247,6 +247,9 @@ func (cs *checkoutService) PlaceOrder(ctx context.Context, req *pb.PlaceOrderReq
 	shippingTrackingAttribute := attribute.String("app.shipping.tracking.id", shippingTrackingID)
 	span.AddEvent("shipped", trace.WithAttributes(shippingTrackingAttribute))
 
+	_, childSpan1 := tracer.Start(ctx, "callPeerService", trace.WithSpanKind(trace.SpanKindClient), trace.WithAttributes(attribute.String("peer.service", "demo-peer-service")))
+	childSpan1.End()
```

Comment:
why not add attribute to the existing span ?

Reply:
Current spans are top level ones. We want to test non top level spans with `peer.service` also getting trace metrics.